.github/ISSUE_TEMPLATE/doc-request.yml

```diff
@@ -0,0 +1,61 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Document
+description: Suggest document changes
+title: "[Doc] "
+labels: ["doc-required"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you very much for submitting suggestions to Pulsar. Please answer the following questions before submitting a doc issue.
```

Comment:
How about adding this for doc? This works for doc as well

<img width="491" alt="image" src="https://user-images.githubusercontent.com/50226895/182801866-7f7ae0e4-c912-4ad5-af07-d84476289fef.png">

Reply:
SGTM. Updating...